Documentation/demo.md

```diff
@@ -0,0 +1,286 @@
+### Demo
+
+Below is an example work flow of using etcd in production.
```

Comment:
"This textual "demo" is a series of examples showing the basic workflow with an etcd cluster."